src/pages/community-stats/community-statistics.styles.scss

```diff
@@ -23,4 +27,4 @@
     flex-direction: column;
     align-items: center;
   }
-}
+}
```

Comment:
I'm not sure but I think our linter has a rule that we need to have an empty line at the end of the file. At least on our other repositories we generally follow that rule. This way this icon doesn't show up:
![imagem](https://user-images.githubusercontent.com/18630253/93687600-ba036d00-fab6-11ea-8384-842541d5c6c5.png)

Could you change it?